briskedu/brisk-learning
# Migrate sessions to DB

Summary:
As we move to multiple machines, we should make sure the session data is stored in the DB instead of in the current server's filesystem.

Test Plan:
1. Confirm session persisted and comes from DB
2. Confirm on reload I can get auth tokens
